source/common/redis/command_splitter_impl.h

```diff
@@ -34,10 +34,15 @@ class CommandHandlerBase {
   ConnPool::Instance& conn_pool_;
 };
 
+class SplitRequestBase : public SplitRequest {
+protected:
+  static void onWrongNumberOfArguments(SplitCallbacks& callbacks, const RespValue& request);
+};
+
 /**
  * SingleServerRequest is a base class for commands that hash to a single backend.
  */
-class SingleServerRequest : public SplitRequest, public ConnPool::PoolCallbacks {
+class SingleServerRequest : public SplitRequestBase, public ConnPool::PoolCallbacks {
```

Comment:
Sorry nevermind I misread what you did. Just switch it back (or move function to static function in unrelated utility class).